h2o-docs/src/product/data-science/algo-params/gainslift_bins.rst

```diff
@@ -0,0 +1,56 @@
+``gainslift_bins``
+------------------
+
+- Available in: GBM, DRF, Naïve-Bayes, XGBoost
+- Hyperparameter: yes
+
+Description
+~~~~~~~~~~~
+
+The `Kolmogorov-Smirnov (KS) <../../performance-and-prediction.html#kolmogorov-smirnov-ks-metric>`__ metric represents the degree of separation between the positive and negative distribution functions for a binomial model. Detailed metrics per each group can be found in the Gains/Lift table. 
+
+The ``gainslift_bins`` option specifies the number of bins for a Gains/Lift table. The default value is ``-1`` and makes the binning automatic. To disable this feature, set to ``0``.
+
+Related Parameters
+~~~~~~~~~~~~~~~~~~
+
+- None
+
+Example
+~~~~~~~
+
+.. tabs::
```

Comment:
The R and Python examples should always match each other. And the values returned should also match in R and Python.